content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
 $ kubectl exec -it $SOURCE_POD -c sleep curl http://httpbin.org/headers
 {{< /text >}}
 
+## Install Istio with access to all the external services by default
+
+An alternative to calling external services directly is to instruct Istio proxy to pass through the calls to all the
+external services. This option allows you to start evaluating Istio quickly, without controlling access to external
+services, and decide to [configure access to external services](#configuring-istio-external-services) later.
+
+Istio has an installation option that allows access to all external services for which no HTTP service exist inside
+the mesh and no HTTP/TCP `ServiceEntry` is defined. For example, if your mesh does not have an HTTP service on port 443,
+and you did not define a `ServiceEntry` on port 443, you can access any external service on port 443. Note, however,
+that once you create an HTTP service on port 443 or define any `ServiceEntry` for any host on
+port 443, all the access to port 443 will be blocked: Istio will fall back to the blocking-by-default behavior for that
+port only. (Defining an HTTP service on port 443 is not recommended anyway, since using the same port for TCP/HTTPS and
+for HTTP traffic in Istio is discouraged.)
+
+1.  Make a couple of requests to external HTTPS services from `SOURCE_POD` to verify that they are blocked (it means you
+    did not configure `ServiceEntry` and `VirtualService` for the external services and that you did not enable
+    direct access to external services).
+
+    {{< text bash >}}
+    $ kubectl exec -it $SOURCE_POD -c sleep -- curl -s https://www.google.com | grep -o "<title>.*</title>"; kubectl exec -it $SOURCE_POD -c sleep -- curl -s https://edition.cnn.com | grep -o "<title>.*</title>"
+    command terminated with exit code 35
+    command terminated with exit code 35
+    {{< /text >}}
+
+1.  To allow access to all the external services, install or update Istio by using
+[Helm](https://preliminary.istio.io/docs/setup/kubernetes/helm-install/) while setting the value of
+`global.outboundTrafficPolicy.mode` to `ALLOW_ANY`: `--set global.outboundTrafficPolicy.mode=ALLOW_ANY`.
+
+    Alternatively, if you followed the instructions in
+    [Quick Start with Kubernetes](https://preliminary.istio.io/docs/setup/kubernetes/quick-start/#installation-steps)
+    and used `install/kubernetes/istio-demo.yaml` or `install/kubernetes/istio-demo-auth.yaml` files to install Istio,
+    just edit the files. Look for the following YAML part:
+
+    {{< text yaml >}}
+        # Set the default behavior of the sidecar for handling outbound traffic from the application:
```

Comment:
Unindent the block content.

Reply:
Fixed by https://github.com/istio/istio.io/pull/3140/commits/f6c5c702809dadd496efc1697b0cee82b132a020